main.go

```diff
@@ -156,10 +156,10 @@ func NewGame(g *Game) {
 			Y: (pathCoord.(map[string]any)["cy"].(float64) + 0.5) * float64(entities.GridSize),
 		}
 	}
-
+	
```

Comment:
Stylistic nitpick, please don't leave trailing white space :P it clutters the git diff.

Reply:
It was not intentional, I remove that